src/arguments/search.py

```diff
@@ -74,21 +75,23 @@ def search_for_results(self, save=False):
         questions = self.utility_object.get_que(json_output)
         if questions == []:
             # evoke an error
-            search_error = SearchError("\U0001F613 No answer found", "\U0001F604 Please try reddit")
+            SearchError("\U0001F613 No answer found",
+                        "\U0001F604 Please try reddit")
         else:
             data = self.utility_object.get_ans(questions)
             print('''
-            \U0001F604 Hopefully you found what you were looking for! 
-            \U0001F4C2 You can save an answer to a file with '-file'
+            \U0001F604  Hopefully you found what you were looking for!
```

Comment:
There seems to be extra an whitespace here, please remove it to keep all interactive emoji dialogues consistent.

Reply:
Oh right! I'll change it.

Reply:
This is a tab space. It helps in aligning text in the terminal. It aligns multiline suggestion "Open browser and post your question..."
![image](https://user-images.githubusercontent.com/26577306/119001723-dd661b00-b9a9-11eb-9e5b-347832e0079d.png)

Should I change it?

Reply:
If it's consistent throughout printing the dialogues, it's totally fine 😄